validator/client/validator.go

```diff
@@ -216,21 +221,15 @@ func (v *validator) UpdateAssignments(ctx context.Context, slot uint64) error {
 				attesterSlot = assignment.Slot
 			}
 			lFields["attesterSlot"] = attesterSlot
-			lFields["proposerSlot"] = "Not proposing"
```

Comment:
why remove these two ?

Reply:
Only deleted shard because it's useless until phase1
`proposerSlot` is moved to below